test-harness/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG base_img_tag
+
```

Comment:
Can you add comments about the usage of this file vs `test-harness/docker/base-images/Dockerfile`?